auraed/src/cri/runtime_service.rs

```diff
@@ -87,31 +88,38 @@ impl runtime_service_server::RuntimeService for RuntimeService {
         todo!()
     }
 
+    /// Run a pod with the Aurae runtime daemon.
     async fn run_pod_sandbox(
         &self,
         request: Request<RunPodSandboxRequest>,
     ) -> Result<Response<RunPodSandboxResponse>, Status> {
+        // Handle Request
         let r = request.into_inner();
+        // Handle Config
         let config = r.config.expect("config from pod sandbox request");
-        let metadata = config.metadata.expect("metadata from config");
-        let windows = config.windows;
+        // Check for Windows config (currently unsupported)
+        let windows = config.clone().windows;
         if windows.is_some() {
-            panic!("Windows architecture is currently unsupported.")
+            panic!("Windows architecture is currently unsupported.") // TODO Unsure if we want to panic here?
```

Comment:
Once we setup validation, I assume this will be a validation error? I guess it depends on what cri clients expect though.